namer/dnssrv/src/integration/scala/io/buoyant/namer/dnssrv/DnsSrvNamerIntegrationTest.scala

```diff
@@ -0,0 +1,22 @@
+package io.buoyant.namer.dnssrv
+
+import com.twitter.finagle.{Addr, Name, NameTree, Path}
+import com.twitter.finagle.stats.NullStatsReceiver
+import com.twitter.util.{Duration, NullTimer}
+import io.buoyant.test.FunSuite
+import org.scalatest.Matchers
+import org.xbill.DNS
+
+class DnsSrvNamerIntegrationTest extends FunSuite with Matchers {
+  test("can resolve some public SRV revord") {
+    val namer = new DnsSrvNamer(Path.empty, new DNS.ExtendedResolver, new NullTimer, Duration.Zero, new NullStatsReceiver)
+    val result = namer.lookupSrv("_http._tcp.mxtoolbox.com.", Path.read("/foo"))
```

Comment:
I think this would be a better black-box test if you use namer.lookup instead.

Reply:
I agree, but I couldn't figure out how to do something like `Future.await()` for an `Activity`. The closest I could find is `Activity.sample`, but I'm not sure if that would race if the lookup is too slow. I'll push something, but please take a look if that's correct.

Reply:
if you import `io.buoyant.namer.RichActivity` then you can do `await(namer.lookup(...).toFuture)`